src/BloomBrowserUI/bookEdit/BookPagesThumbnailList/BookPagesThumbnailList.htm

```diff
@@ -31,12 +31,36 @@
             jQuery('.gridItem').click(function () {
                 fireCSharpEvent("gridClick", $(this).attr('id'));
             });
+
+            setTimeout(loadNextThumbnail, 100);
```

Comment:
In a comment below you talked about reducing a timeout to 20ms, but this one at least is still 100. Is there any sort of basis for it? For example, are we wanting to wait about as long as it takes to process the longest request? The shortest? 1/threadcount of one of those? Long enough to give the GC or the UI or the Idle or some Gecko thread a chance? If we're trying to prevent processing too many images at the same time, would it be better to just have fewer worker threads? Even if the basis is "it just works" it would be helpful to document what experiment indicates that it just works and should be retried if someone wants to change it.